include/nuttx/fs/fs.h

```diff
@@ -405,6 +405,7 @@ struct inode
   uint16_t          i_flags;    /* Flags for inode */
   union inode_ops_u u;          /* Inode operations */
   ino_t             i_ino;      /* Inode serial number */
+  size_t            i_size;     /* The size of per inode driver */
```

Comment:
If CONFIG_PSEUDOFS_FILE is not set, is `i_size` needed?

Reply:
i_size is used to store size information of any inode type, not just pseduo file? @davids5

Reply:
Maybe we can have private in `struct fs_pseudofile_s`?

Reply:
since fstat callback exist only in mountpt_operations, not file_operations, it's hard to return it to caller if i_size is moved into fs_pseudofile_s.

Reply:
add CONFIG_PSEUDOFS_FILE  for i_size in struct inode